src/python/WMCore/Agent/Database/CreateAgentBase.py

```diff
@@ -59,10 +59,10 @@ def __init__(self, logger = None, dbi = None, params = None):
              pid           INTEGER,
              poll_interval INTEGER      NOT NULL,
              last_error    INTEGER,
-             cycle_time    DECIMAL(9,4) DEFAULT 0 NOT NULL,
+             cycle_time    FLOAT DEFAULT 0 NOT NULL,
              outcome       VARCHAR(1000),
              error_message VARCHAR(1000),
-             UNIQUE (component_id, name))"""
+             UNIQUE (name))"""
```

Comment:
It covers the case if different components has the same worker name, but we don't have that case anyway. 😄